src/app/app.component.html

```diff
@@ -1,4 +1,4 @@
-<!--The content below is only a placeholder and can be replaced.-->
+<app-nav></app-nav>
```

Comment:
Why did you add navigation?

Reply:
Navigation is the place for our menu, so users can select which page to go. 
Like 
Home . PlayList . About . Contact